apps/calendar/js/backend/calendar_worker.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
I split the service from the worker file to keep things nice and clean (this way the service can declare its dependencies with the synchronous-looking require call)